.github/workflows/build.yaml

```diff
@@ -0,0 +1,41 @@
+name: Docker Image
```

Comment:
should you name this workflow docker?

Reply:
For sure! still fully WIP and with lots of bad naming just trying to make it work ^^

Reply:
I removed the new files and commented about it here: https://github.com/lambdaclass/lambda_ethereum_rust/pull/1032#issuecomment-2450670401